web/ui/react-app/src/pages/Rules.tsx

```diff
@@ -1,15 +1,108 @@
 import React, { FC } from 'react';
 import { RouteComponentProps } from '@reach/router';
 import PathPrefixProps from '../PathPrefixProps';
-import { Alert } from 'reactstrap';
-
-const Rules: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => (
-  <>
-    <h2>Rules</h2>
-    <Alert color="warning">
-      This page is still under construction. Please try it in the <a href={`${pathPrefix}/rules`}>Classic UI</a>.
-    </Alert>
-  </>
-);
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { Alert, Table } from 'reactstrap';
+import { useFetch } from '../utils/useFetch';
+import { Link } from '@reach/router';
+
+interface Rule {
+  name: string;
+  query: string;
+  health: string;
+  evaluationTime: string;
+  lastEvaluation: string;
+  type: string;
+  lastError: string;
+}
+
+interface EnrichRule {
```

Comment:
This does not seem to refer to an enriched rule, but to a group of rules, so it should probably be:

```suggestion
interface RuleGroup {
```